cms/templates/cms/toolbar/dragitem_extra_menu.html

```diff
@@ -1,3 +1,3 @@
 {% for item in items %}
-<div class="cms-submenu-item"><a href="{{ item.url }}" data-rel="{{ item.action }}"{% if item.data %} data-post="{{ item.data }}"{% endif %}{% if question %} data-text="{{ question }}"{% endif %} data-on-success="REFRESH_PAGE">{{ item.name }}</a></div>
+<div class="cms-submenu-item"><a href="{{ item.url }}" {% for key, attr in item.attributes.items %}data-{{ key }}="{{ attr }}" {% endfor %} data-rel="{{ item.action }}"{% if item.data %} data-post="{{ item.data }}"{% endif %}{% if question %} data-text="{{ question }}"{% endif %} data-on-success="REFRESH_PAGE">{{ item.name }}</a></div>
```

Comment:
potentially possible to override other data attributes we already have, like `post` or `text` since jquery will take the first occurrence, not sure if we really want that or not